1-exercises/B-object-arrays/exercise-1.js

```diff
@@ -14,7 +14,10 @@ let people = [
 ];
 
 let peopleNames =
-  /* todo */
+  [];
+  for(let names of people){
+    peopleNames.push(names.name)
+  }
```

Comment:
.map() method can be very handy in this case. please try to do this exercise **using.map()**
The hint is :
people.map() .....